jcommune-model/src/test/java/org/jtalks/jcommune/model/entity/JCUserTest.java

```diff
@@ -217,6 +220,37 @@ public void spanishLanguageHandling () {
         assertEquals(language, Language.ENGLISH);
     }
 
+    @Test
+    public void entityFieldsShouldBeSerialized() throws NoSuchMethodException, InvocationTargetException,
+            IllegalAccessException {
+        JCUser user = ObjectsFactory.getUserWithAllNotInheritedFieldsFilled();
+        user.setAvatar(new byte[]{1});
```

Comment:
We still fill them even if we fill them in tests :) I prefer to move them to `ObjectsFactory` to keep the tests as readable as possible.